docs/sources/configuration/_index.md

```diff
@@ -1961,18 +1997,22 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 [max_streams_per_user: <int> | default = 0]
 
 # Maximum line size on ingestion path. Example: 256kb.
-# There is no limit when unset.
+# There is no limit when unset or set to 0.
 # CLI flag: -distributor.max-line-size
-[max_line_size: <string> | default = none ]
+[max_line_size: <string> | default = 0 ]
 
 # Truncate log lines when they exceed max_line_size.
 # CLI flag: -distributor.max-line-size-truncate
-[max_line_size_truncate: <boolean> | default = false ]
+[max_line_size_truncate: <bool> | default = false ]
 
 # Maximum number of log entries that will be returned for a query.
 # CLI flag: -validation.max-entries-limit
 [max_entries_limit_per_query: <int> | default = 5000 ]
 
+# Maximum number of active streams per user, per ingester. 0 to disable.
+# CLI flag: -ingester.max-streams-per-user
+[max_streams_per_user: <int> | default 0]
```

Comment:
I don't understand what the default value of 0 causes to happen.  What does it disable?  I'm guessing that the value of 0 eliminates a cap.

Reply:
I'm not sure either, can try to find out later looking through the code

Reply:
it was easy to find, 0 effectively makes it unlimited. I've rephrased that as well 
https://github.com/grafana/loki/blob/4d8009cacc28cd15567b8af481576ec2f54f1c7b/pkg/ingester/limiter.go#L94